src/Abstractions/NexusMods.Abstractions.Loadouts/Loadout.cs

```diff
@@ -119,6 +122,20 @@ public LoadoutWithTxId GetLoadoutWithTxId()
         /// especially, when it comes to displaying elements the user can edit.
         /// </remarks>
         public bool IsVisible() => LoadoutKind == LoadoutKind.Default;
+        
+        
+        /// <summary>
+        /// Returns the timestamp of the transaction that created this loadout.
+        /// </summary>
+        public DateTime CreatedAt
+        {
+            get
+            {
+                var lowestTx = this.Min(d => d.T);
+                var tx = Transaction.Load(Db, EntityId.From(lowestTx.Value));
+                return tx.Timestamp;
+            }
+        }
     }
```

Comment:
We have an extension method for this: `GetCreatedAt`

Reply:
Should I remove Mod.CreatedAt as well?

Reply:
Removed